alws/constants.py

```diff
@@ -6,10 +6,11 @@
 
 __all__ = ['BuildTaskStatus', 'ReleaseStatus', 'TestTaskStatus',
            'BuildTaskRefType', 'SignStatus', 'RepoType', 'ExportStatus',
-           'debuginfo_regex', 'REQUEST_TIMEOUT']
+           'debuginfo_regex', 'REQUEST_TIMEOUT', 'DRAMATIQ_TASK_TIMEOUT']
 
 
 REQUEST_TIMEOUT = 60  # 1 minute
+DRAMATIQ_TASK_TIMEOUT = 3600000  # 1 hour in milliseconds
```

Comment:
Just for future: I guess better to write multiplication of some constants like this: 60*60*1000
It's more readable